run.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+echo "Welcome to Unity Catalog Guided Setup"
+echo "#######################################"
+
+
+confirm() {
+  echo "Please confirm before continuing (y/n)".
+  read -r confirmed
+  if [[ $confirmed == 'n' ]]; then
+    exit 2
+  fi
+}
+
+#Setup installers
+if [[ $OSTYPE == "linux-gnu"* ]]; then
+      echo "This tool will ensure that required dependencies are installed. Your permissions may be elevated via sudo during this process and you may be prompted, please do not run this command as root."
+      confirm;
+      if command -v yum &> /dev/null
+      then
+         sudo yum install -y wget unzip
+      fi
+      if command -v apt-get &> /dev/null
+      then
+        sudo apt-get install wget unzip
+      fi
+      if ! command -v terraform &> /dev/null
+      then
+        sudo wget https://releases.hashicorp.com/terraform/1.1.3/terraform_1.1.3_linux_amd64.zip
+        sudo unzip ./ terraform_0.12.2_linux_amd64.zip –d /usr/local/bin
```

Comment:
the version does not match the one from wget

Reply:
updated